Orchestrator.py

```diff
@@ -137,6 +137,7 @@ def __init__(self, partitionTargetValue, dynamoDBRegion, scalingProfile, dryRun=
 		try:
 			self.dynDBR = boto3.resource('dynamodb', region_name=self.dynamoDBRegion)
 			self.tierSpecTable = self.dynDBR.Table(self.tierSpecTableName)
+			self.WorkloadStateTable = self.dynDBR.Table('WorkloadState')
```

Comment:
Create a ORCHESTRATOR.<workload state table name> instead of hard coding it here.